synapse/tests/test_model_inet.py

```diff
@@ -70,6 +70,15 @@ def test_model_inet_asnet4(self):
             self.nn(core.getTufoByProp('inet:ipv4', 0x01020304))
             self.nn(core.getTufoByProp('inet:ipv4', 0x05060708))
 
+            t1 = core.formTufoByProp('inet:asn', 12345)
+            self.none(t1[1].get('inet:asn:owner'))
+            t1 = core.setTufoProps(t1, owner='vertex')
+            self.eq(t1[1].get('inet:asn:owner'), 'vertex')
+            # TODO: Uncomment when we have a global alias resolver in place.
+            # self.nn(core.getTufoByProp('ou:alias', 'vertex'))
+            t2 = core.formTufoByProp('inet:asn', 12346, owner='vertex')
```

Comment:
make owner ou:org ( and access via $alias if needed )

Reply:
fixed